### ISSUE TITLE
git-clone.coffee::get_repo_name: returns incomplete repo_name

### ISSUE THREAD

**Issue by omelhoro** [brianchung808/git-clone#1]
There are these lines. I wonder why we have to split and join the name. Cloning something like
https://github.com/tastejs/todomvc

worked only after I outcommented the last two lines

``` javascript

# parse out repo name from url
get_repo_name = (repo_uri) ->
  tmp = repo_uri.split('/')
  repo_name = tmp[tmp.length-1]
  # why we split and join here?
  tmp = repo_name.split('.')
  repo_name = tmp[...-1].join('.')
```

**Comment by brianchung808** [comment]
Haven't looked at this in a while, but it seems my (poor) assumption is that the url being used ends with a `.git`, which explains the the last 2 lines.

**Comment by omelhoro** [comment]
Actually you're partly right: When cloning per ssh the ending is `.git`, when per https there is no.

**Comment by brianchung808** [comment]
Specifying `.git` is valid for both https and ssh. Copying the clone URL from the right panel on github for HTTPS has the ending.

Anyways, more people would probably copy the url from the browser search bar so urls without the ending should also be supported.

**Comment by brianchung808** [comment]
Fixed in #2. In `v0.1.5`

**Comment by omelhoro** [comment]
Thanks!))
